build-system/common/check-package-manager.js

```diff
@@ -331,18 +339,21 @@ function checkPythonVersion() {
         cyan(recommendedVersion) + yellow('.')
       );
       console.log(
-        yellow('⤷ To fix this, install the correct version from'),
-        cyan(`https://www.python.org/download/releases/${recommendedVersion}`) +
-          yellow('.')
+        yellow('⤷ To fix this, install a supported version from'),
+        cyan('https://www.python.org/downloads') + yellow('.')
```

Comment:
Add a trailing `/` after `downloads` to make the url `'https://www.python.org/downloads/'`. Otherwise, the trailing period will cause the link to 404.